File: ML/Coderaman-tech_AmanKumarShrivastava_2024cse1155_2/GettingStarted/grading_system.cpp
#include<iostream>
using namespace std;
int main(){
    int marks;
    cin>>marks;
    
   if(marks>90)
   cout<<"excellent"<<"\n";
   else if(marks>80 && marks<=90)
   cout<<"good"<<"\n";
   else if(marks>70 && marks<=80)
   cout<<"fair"<<"\n";
   else if(marks>60 && marks<=70)
  cout<<"meets expectations"<<"\n";
   else 
   cout<<"below par"<<"\n";
}

File: ML/SparshB21_SparshBansal_2024it1109_2/GettingStarted/GradingSystem.java
import java.util.*;
public class GradingSystem{
  
    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        int marks = sc.nextInt();
           if(marks>90)
           System.out.println("Excellent");
           else if(marks<=90 && marks>80)
           System.out.println("Good");
           else if(marks<=80 && marks>70)
           System.out.println("Fair");
           else if(marks<=70 && marks>60)
           System.out.println("Meets Expectations");
           else if(marks<=60)
           System.out.println("Below Par");     
           sc.close();
        }
}